kornia/augmentation/random_generator/_2d/crop.py

```diff
@@ -186,15 +186,15 @@ def __repr__(self) -> str:
         repr = f"scale={self.scale}, resize_to={self.ratio}, output_size={self.output_size}"
         return repr
 
-    def make_samplers(self, device: torch.device, dtype: torch.dtype) -> None:
-        scale = torch.as_tensor(self.scale, device=device, dtype=dtype)
-        ratio = torch.as_tensor(self.ratio, device=device, dtype=dtype)
+    def make_samplers(self, device: Device = None, dtype: Dtype = None) -> None:
+        scale = as_tensor(self.scale, device=device, dtype=dtype)
```

Comment:
```suggestion
        scale = tensor(self.scale, device=device, dtype=dtype)
```
to be consistent with rest of the library? not quite sure about the benefits and more aligned with numpy.ndarray constructor

Reply:
probably, scale is already a tensor, so is just to guarantee the right device/type